pages/header.py

```diff
@@ -0,0 +1,20 @@
+from methods import Methods
+from locators import Locators
+
+class Header():
+    """This is parent class"""
```

Comment:
actually, it is not a parent page. it is just page for header

Reply:
Ok, I'll change